canonical/canonical.com
# Update text on homepage

## Done

- Update text on homepage as per [copydoc](https://docs.google.com/document/d/1GNa1FaTkARdrVwYgHXf5H-6sqOLjkJENDistDDcsi4o/edit)

## QA

- Go to the [home page](https://canonical-com-1238.demos.haus/) and see the text has been updated
Before:
![image](https://github.com/canonical/canonical.com/assets/58276363/c90ab0cb-53d3-4ca8-995e-8da9979f566c)
After:
![image](https://github.com/canonical/canonical.com/assets/58276363/b4cbe797-b3e7-4bcb-bc0e-2786d45fa774)


## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10371
